apps/wallet-mobile/src/features/Receive/common/useMultipleAddressesInfo.tsx

```diff
@@ -37,7 +37,7 @@ const useIsShowingMultipleAddressesInfo = (options?: UseQueryOptions<boolean, Er
     },
   })
 
-  return query.data
+  return query.isFetching ? false : query.data
```

Comment:
hmm, this looks that should be handled by `suspense` can't we leverage `suspense` ?